LanguageFeatures/Wildcards/imports_lib2.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Import prefixes named `_` are non-binding and will provide access
+/// to the extensions in that library.
+///
+/// @description Checks that an import prefix named `_` is non binding and
+/// provides access to the imported library.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=wildcard-variables
+
+int one = 1;
```

Comment:
We should again have an extension here.

It would probably be a good idea to have an extension conflict, such that we can do `42.foo()` and get an error because `foo` is ambiguous on `int`, because we have an extension from `lib1` as well as an extension from `lib2`.